scripts/get_baseline_hash.py

```diff
@@ -21,7 +22,8 @@ def filter_results(issue):
         if label['name'] in filter_labels:
             labels_check = False
             break
-    title_check = "Testsuite Status" in issue["title"]
+    regex_check = re.search("^Testsuite Status.*$", issue["title"]) # if fails, regex_check == None
```

Comment:
I think we can get away with this to make the comparison a single statement:
```
    title_check = re.search("^Testsuite Status", issue["title"]) is not None # if the regex search fails, re.search returns None
```

Reply:
this also works